src/styles/variables/_icons.scss

```diff
@@ -11,5 +11,11 @@ $icon-arrow-link: "\f0a9";
 $icon-asterisk: "\f069";
 $icon-check: "\F00C";
 $icon-home: "\f015";
-$icon-site-nav-default-size:30px;
```

Comment:
With line 14 and 15 being removed, confirming, all properties using `$icon-site-nav-default-size` and `$icon-arrow-default-size` have been addressed. Disregard if these were unused variables.